src/shared/components/modlog.tsx

```diff
@@ -911,9 +911,11 @@ export class Modlog extends Component<ModlogRouteProps, ModlogState> {
               <>
                 <Link
                   className="text-body"
-                  to={`/c/${communityResp.community_view.community.name}`}
+                  to={
+                    communityLink(communityResp.community_view.community).link
+                  }
                 >
-                  /c/{communityResp.community_view.community.name}
+                  {communityLink(communityResp.community_view.community).link}
```

Comment:
Dont see any way to create a variable here, and doesnt seem worth the effort making a separate function that is only used once.

Reply:
That's fine. Its pretty annoying to do a lot of variable manipulation inside react html, and the only way to do it correctly is to extract to a function / component.

But also, you can probably just use `<CommunityLink` here anyway.